tools/generator_python3.py

```diff
@@ -0,0 +1,216 @@
+#!/usr/bin/env python
+# coding=ascii
+###############################################################################
+# Copyright 2015-2016 University of Florida. All rights reserved.
```

Comment:
Please update any copyright to read 2015-2019 for any files you add or change. Do not change files that are unrelated.